server/zanata-war/src/main/webapp/project/project.xhtml

```diff
@@ -231,6 +231,9 @@
         </li>
       </ui:fragment>
       <li>
+        <ui:fragment
+            rendered="#{not empty projectHome.instance.homeContent or
+            identity.hasPermission(projectHome.instance, 'update')}">
```

Comment:
Should the maintainer be able to see that their project has no content there? Or even a link to the About setting?
If not, I can just remove it for all.